From: Andreas Henriksson <andreas@fatal.se>
Subject: Disable flaky textview-margins reftest

This test does not fail for me when building locally and neither on when tested
on "deb-o-matic". Only when built on debian buildd.

13:36 < ah> Hi. I'm having problems with the textview-margins.ui reftest ... it works 
            when I build locally but since after 3.19.12 it now fails on debian 
            buildds (i.e. 3.20.0 and 3.20.1 fails, while 3.19.12 built fine). Anyone 
            got any ideas why?
13:40 < baedert> ah: that one failed after a commit from me, but mclasen fixed it 
                 later, but it looks like that didn't work out

Given that the test seems to be known flaky, disable it for now.

--- a/testsuite/reftests/Makefile.am
+++ b/testsuite/reftests/Makefile.am
@@ -426,9 +426,6 @@
 	textview-border-windows.css \
 	textview-border-windows.ref.ui \
 	textview-border-windows.ui \
-	textview-margins.css \
-	textview-margins.ref.ui \
-	textview-margins.ui \
 	textview-tags.ref.ui \
 	textview-tags.ui \
 	toplevel-vs-popup.ref.ui \
--- a/testsuite/reftests/textview-margins.ui
+++ /dev/null
@@ -1,26 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<!-- Generated with glade 3.18.3 -->
-<interface>
-  <requires lib="gtk+" version="3.0"/>
-  <object class="GtkTextBuffer" id="textbuffer1">
-    <property name="text" translatable="yes">Lorem ipsum dolor sit amet, consectetur adipisci elit, sed eiusmod tempor incidunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquid ex ea commodi consequat. Quis aute iure reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint obcaecat cupiditat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum</property>
-  </object>
-  <object class="GtkWindow" id="window">
-    <property name="can_focus">False</property>
-    <property name="type">popup</property>
-    <property name="default_width">106</property>
-    <property name="resizable">0</property>
-    <child>
-      <object class="GtkTextView" id="textview1">
-        <property name="visible">True</property>
-        <property name="can_focus">True</property>
-        <property name="wrap_mode">word</property>
-        <property name="margin_start">2</property>
-        <property name="margin_end">2</property>
-        <property name="left_margin">8</property>
-        <property name="right_margin">8</property>
-        <property name="buffer">textbuffer1</property>
-      </object>
-    </child>
-  </object>
-</interface>
--- a/testsuite/reftests/textview-margins.css
+++ /dev/null
@@ -1 +0,0 @@
-@import "reset-to-defaults.css";
--- a/testsuite/reftests/textview-margins.ref.ui
+++ /dev/null
@@ -1,26 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<!-- Generated with glade 3.18.3 -->
-<interface>
-  <requires lib="gtk+" version="3.0"/>
-  <object class="GtkTextBuffer" id="textbuffer1">
-    <property name="text" translatable="yes">Lorem ipsum dolor sit amet, consectetur adipisci elit, sed eiusmod tempor incidunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquid ex ea commodi consequat. Quis aute iure reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint obcaecat cupiditat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum</property>
-  </object>
-  <object class="GtkWindow" id="window">
-    <property name="can_focus">False</property>
-    <property name="type">popup</property>
-    <property name="default_width">106</property>
-    <property name="resizable">0</property>
-    <child>
-      <object class="GtkTextView" id="textview1">
-        <property name="visible">True</property>
-        <property name="can_focus">True</property>
-        <property name="wrap_mode">word</property>
-        <property name="margin_start">8</property>
-        <property name="margin_end">8</property>
-        <property name="left_margin">2</property>
-        <property name="right_margin">2</property>
-        <property name="buffer">textbuffer1</property>
-      </object>
-    </child>
-  </object>
-</interface>
